TargetComposition/SymBuilder/HelpStrings.rc

```diff
@@ -55,4 +56,20 @@ BEGIN
     SYMBOLBUILDER_IDS_GLOBALDATA_TYPE               "The type of the global data"
     SYMBOLBUILDER_IDS_GLOBALDATA_OFFSET             "The offset of the global data within its loaded module"
     SYMBOLBUILDER_IDS_GLOBALDATA_DELETE             "Delete() - Deletes the global data"
+    SYMBOLBUILDER_IDS_FUNCTIONS_CREATE              "Create(name, returnType, codeOffset, codeSize, [qualifiedName], [parameter]...) - Creates a new global function with the specified return type and code range.  Parameters are added separately through API calls on the returned object"

```

Comment:
>Parameters are added separately through API calls on the returned object [](http://example.com/codeflow?start=213&length=72)

Is "Parameters are added separately through API calls on the returned object" accurate? Since it looks like you can put them at the end here.

Reply:
I can fix that...  I added the optional argument *AFTER* writing the help string.  You can do it either way.

Reply:
Fixed in iteration 2.